### PR TITLE
AIOps data quality checker

### PR DESCRIPTION
**Pull request by maxhwchang**
Signed-off-by: Hau-wen Chang <changha@us.ibm.com>

The PR is to comply with the opensource for the AIOps log data quality checker.

### CODE REVIEW

File: aiops-log-data-quality-checker/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+to look for and tackle low-hanging fruit in the [issue tracker](https://github.com/ibm/aiops-log-data-checker/issues).
+Before embarking on a more ambitious contribution, please quickly get in touch with us.
+
+**Note: We appreciate your effort, and want to avoid a situation where a contribution
+requires extensive rework (by you or by us), sits in backlog for a long time, or
+cannot be accepted at all!**
+
+### Proposing new features
+
+If you would like to implement a new feature, please [raise an issue](https://github.com/ibm/aiops-log-data-checker/issues)
```
Comment (neil-boyette-ibm) [inline aiops-log-data-quality-checker/CONTRIBUTING.md:17]: Please update to this repo.

File: aiops-log-data-quality-checker/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+before sending a pull request so the feature can be discussed. This is to avoid
+you wasting your valuable time working on a feature that the project developers
+are not interested in accepting into the code base.
+
+### Fixing bugs
+
+If you would like to fix a bug, please [raise an issue](https://github.com/ibm/aiops-log-data-checker/issues) before sending a
```
Comment (neil-boyette-ibm) [inline aiops-log-data-quality-checker/CONTRIBUTING.md:24]: Please update to this repo

File: aiops-log-data-quality-checker/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+You can include this automatically when you commit a change to your
+local git repository using the following command:
+
+```
+git commit -s
+```
+
+## Communication
+Please feel free to connect with us on our [Slack channel](https://ibm-analytics.slack.com/archives/CL3DQ80EN).
```
Comment (neil-boyette-ibm) [inline aiops-log-data-quality-checker/CONTRIBUTING.md:72]: This is an internal slack channel. Let's replace it w/ a link to the maintainers md file.

File: aiops-log-data-quality-checker/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+## Contributing
+Pull requests are very welcome! Make sure that your patches are tested. Ideally create a topic branch for every separate change you make. For example:
+
+1. Fork the repo
+2. Create your feature branch (`git checkout -b my-new-feature`)
+3. Commit your changes (`git commit -am 'Added some feature'`)
+4. Push to the branch (`git push origin my-new-feature`)
+5. Create new Pull Request
+
+**Note:** Make sure you update the [Changelog](CHANGELOG.md) each time you add, remove, or update sample files.
```
Comment (neil-boyette-ibm) [inline aiops-log-data-quality-checker/README.md:53]: Please add a reference to the contributing md file for more details.

File: aiops-log-data-quality-checker/logs_data_checker_tests/data/normalized_example.json [hunk 5]
```diff
@@ -0,0 +1 @@
+{"message":"invoking [applyModel] WorkspaceId = 1b69b204-5432-496f-8e61-da7e4b037730","timestamp":1578669135680,"instance_id":"nlu","features":[],"meta_features":[{"type":"WindowMeta","obj_value":{"start":1578669130000,"end":1578669140000}}],"entities":{"pod":"pmp015-nlu-6bc9cb9574-pgkgg","_cluster":"ld60","container":"nlu"},"application_group_id":"1","application_id":"1"}
```
Comment (neil-boyette-ibm) [inline aiops-log-data-quality-checker/logs_data_checker_tests/data/normalized_example.json:1]: Please replace w/ generic example file

File: aiops-log-data-quality-checker/logs_data_checker_tests/data/test_line.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
+{
+  "message": "invoking [applyModel] WorkspaceId = 1b69b204-5432-496f-8e61-da7e4b037730",
+  "timestamp": 1578669135680,
+  "instance_id": "nlu",
```
Comment (neil-boyette-ibm) [inline aiops-log-data-quality-checker/logs_data_checker_tests/data/test_line.json:4]: Please replace w/ generic test file

File: aiops-log-data-quality-checker/logs_data_checker_tests/data/test_main_bad.input [hunk 7]
```diff
@@ -0,0 +1,12 @@
+{"application_group_id": "1", "application_id": "1", "application": "node1", "timestamp": 1591039175920.0, "utc_timestamp": "2020-06-01T19:19:35.920000Z", "type": "log", "instance_id": "node1", "message": " ALL 000000000000 GLOBAL_SCOPE FtpSession.isCommandAllowed() There is no command limiting policy defined for this IP/user", "features": [], "entities": {}, "meta_features": [{"type": "WindowMeta", "obj_value": {"start": 1591039170000, "end": 1591039180000}}]}
```
Comment (neil-boyette-ibm) [inline aiops-log-data-quality-checker/logs_data_checker_tests/data/test_main_bad.input:1]: Please replace w/ generic example file

File: aiops-log-data-quality-checker/logs_data_checker_tests/data/test_main_good.input [hunk 8]
```diff
@@ -0,0 +1,10 @@
+{"application_id": "def456", "entities": {"pod": "pmp026-nlu-5d77786646-676q6", "_cluster": "ld60", "container": "nlu"}, "features": [], "instance_id": "nlu", "level": 1, "message": "invoking [applyModel] WorkspaceId = 426e479d-3e80-426b-a4ee-adeee3496670", "meta_features": [{"type": "WindowMeta", "obj_value": {"start": 1565481540000, "end": 1565481550000}}], "application_group_id": "1", "type": "StandardLog", "utc_timestamp": "2019-08-10 23:59:04.092000", "timestamp": 1565481544092}
```
Comment (neil-boyette-ibm) [inline aiops-log-data-quality-checker/logs_data_checker_tests/data/test_main_good.input:1]: Please replace w/ generic example file

**Review by neil-boyette-ibm — CHANGES_REQUESTED** [review]
Once you address the changes, I'll approve

File: aiops-log-data-quality-checker/README.md (hunk 4 above)
Comment (maxhwchang) [reply aiops-log-data-quality-checker/README.md:55]: done!

File: aiops-log-data-quality-checker/CONTRIBUTING.md (hunk 1 above)
Comment (maxhwchang) [reply aiops-log-data-quality-checker/CONTRIBUTING.md]: done

File: aiops-log-data-quality-checker/CONTRIBUTING.md (hunk 3 above)
Comment (maxhwchang) [reply aiops-log-data-quality-checker/CONTRIBUTING.md]: done!

File: aiops-log-data-quality-checker/logs_data_checker_tests/data/normalized_example.json (hunk 5 above)
Comment (maxhwchang) [reply aiops-log-data-quality-checker/logs_data_checker_tests/data/normalized_example.json]: done

File: aiops-log-data-quality-checker/logs_data_checker_tests/data/test_main_bad.input (hunk 7 above)
Comment (maxhwchang) [reply aiops-log-data-quality-checker/logs_data_checker_tests/data/test_main_bad.input]: done

File: aiops-log-data-quality-checker/logs_data_checker_tests/data/test_line.json (hunk 6 above)
Comment (maxhwchang) [reply aiops-log-data-quality-checker/logs_data_checker_tests/data/test_line.json]: done

File: aiops-log-data-quality-checker/README.md (hunk 4 above)
Comment (maxhwchang) [reply aiops-log-data-quality-checker/README.md:55]: done

File: aiops-log-data-quality-checker/logs_data_checker_tests/data/test_main_good.input (hunk 8 above)
Comment (maxhwchang) [reply aiops-log-data-quality-checker/logs_data_checker_tests/data/test_main_good.input]: done

File: aiops-log-data-quality-checker/CONTRIBUTING.md (hunk 2 above)
Comment (maxhwchang) [reply aiops-log-data-quality-checker/CONTRIBUTING.md]: done!

**Review by neil-boyette-ibm — APPROVED** [review]
LGTM - Thanks Max !

**Pull request merged**